lib/Runtime/Library/JSONStack.cpp

```diff
@@ -24,10 +33,18 @@ namespace JSON
     {
         if (bJsObject)
         {
+            if (STACK_VALUE)
+            {
+                return (data == STACK_VALUE);
```

Comment:
>return (data == STACK_VALUE); [](start = 16, length = 29)

just to guard future changes to this class, can you add assert at appropriate places to check when STACK_VALUE != nullptr, jsobjectStack is always empty and similarly for dom stack?